src/core/structures.hpp

```diff
@@ -7,14 +7,14 @@ namespace core {
 
 struct User {
   std::string userID;
-  int price;
+  double price;
```

Comment:
if there is no `double` in oatpp then maybe we could stick to `float` here also

Reply:
@TStypulkowski needs to change this in the database